sdk/batch/arm-batch/test/sample.env

```diff
@@ -0,0 +1,30 @@
+Test Configuration Instructions
+================================
+
+
+In Live Mode
+-------------
+
+- First create a Service Principal for the subscription you wish to test against.
+  Follow the instructions here on creating a Service Principal using the Azure CLI:
+  https://github.com/Azure/azure-sdk-for-node/blob/master/Documentation/Authentication.md
+
+- Once the Service Principal is configured, set the following env variables.
+
+    AZURE_SUBSCRIPTION_ID:  Azure subscription GUID
+    AZURE_CLIENT_ID: The Service Principal Application ID
+    AZURE_CLIENT_SECRET: The password configured for the AAD Application
+    AZURE_TENANT_ID: Tenant (e.g. microsoft.com)
+    
+- The tests are configured by default to run with in japaneast, using a resource group of
+  'default-azurebatch-japaneast'. To change this location set:
+    
+    AZURE_TEST_LOCATION: Location name
+    
+- The tests require a storage account to be configured. Using the CLI:
+    >> az storage account create -n <name> -l <location> -g <resource group> --sku Standard_LRS
+    
+   The tests have been configured against a <storage account name> of nodesdkteststorage. If you have set up
+   a different name, then set:
+   
+    AZURE_AUTOSTORAGE: Your storage account name
```

Comment:
Thanks for adding the file! However, the file format should be as follows:
```
# This is a comment
ENV1=<some default val or a description>
...
```
so I would suggest make that file a readme file and create another `sample.env` that has the following:
```
# First create a Service Principal for the subscription you wish to test against.
# Follow the instructions here on creating a Service Principal using the Azure CLI:
# https://github.com/Azure/azure-sdk-for-node/blob/master/Documentation/Authentication.md
AZURE_SUBSCRIPTION_ID=<Azure subscription GUID>
AZURE_CLIENT_ID=<The Service Principal Application ID>
AZURE_CLIENT_SECRET=<The password configured for the AAD Application>
AZURE_TENANT_ID=<Tenant (e.g. microsoft.com)>
# The tests are configured by default to run with in japaneast, using a resource group of
# 'default-azurebatch-japaneast'. To change this location set:
AZURE_TEST_LOCATION="default-azurebatch-japaneast"
# The tests require a storage account to be configured. Using the CLI:
#   >> az storage account create -n <name> -l <location> -g <resource group> --sku Standard_LRS
# The tests have been configured against nodesdkteststorage. If you have set up
#  a different name, then set this var:
AZURE_AUTOSTORAGE="nodesdkteststorage"
```